### PR TITLE
WEBUI-841: createLayouts for UserWorkspaces

### PR DESCRIPTION
**Pull request by GuruprasadAnk1996**
https://jira.nuxeo.com/browse/WEBUI-841

### CODE REVIEW

File: elements/document/userworkspacesroot/nuxeo-userworkspacesroot-edit-layout.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-workspaceroot-create-layout`
```
Comment (rakeshkumar1019) [inline elements/document/userworkspacesroot/nuxeo-userworkspacesroot-edit-layout.html]: file name and commented file name is different.

File: elements/document/userworkspacesroot/nuxeo-userworkspacesroot-metadata-layout.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-workspace-metadata-layout`
+@group Nuxeo UI
+@element nuxeo-workspace-metadata-layout
```
Comment (rakeshkumar1019) [inline elements/document/userworkspacesroot/nuxeo-userworkspacesroot-metadata-layout.html]: here also, names in commented code & current file name is different.

File: elements/document/userworkspacesroot/nuxeo-userworkspacesroot-view-layout.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-workspace-view-layout`
+@group Nuxeo UI
+@element nuxeo-workspace-view-layout
```
Comment (rakeshkumar1019) [inline elements/document/userworkspacesroot/nuxeo-userworkspacesroot-view-layout.html]: same here also, see[ here](https://github.com/nuxeo/nuxeo-web-ui/pull/1539/files#r930985817)

**Review by rakeshkumar1019 — CHANGES_REQUESTED** [review]
see above comments

**Comment by semisse** [comment]
Please, also review your commits, since I think this can be in one single commit.

**Comment by semisse** [comment]
Another thing: don't forget in the commit messages we use words spaced, like a regular sentence

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good!

**Pull request closed without merging**